lib/miq_tools_services/minigit.rb

```diff
@@ -153,19 +144,11 @@ def mergeable?(branch = nil, into_branch = "master")
       end
     end
 
-    def update_pr_branch(branch = nil, remote = "upstream")
-      create_or_update_pr_branch(branch || current_branch, remote)
-    end
-
-    def create_pr_branch(branch, remote = "upstream")
-      create_or_update_pr_branch(branch, remote)
-    end
-
-    private
-
-    def create_or_update_pr_branch(branch, remote)
-      fetch("-fu", remote, "refs/pull/#{pr_number(branch)}/head:#{branch}")
-      reset("--hard")
+    def ensure_fetch_refs
+      remote.split("\n").uniq.compact.each do |remote|
+        fetches = config("--get-all", "remote.#{remote}.fetch").split("\n").compact
```

Comment:
Similarly, this can be a method `def fetches(remote); config("--get-all", "remote.#{remote}.fetch").split("\n").compact; end`